### PR TITLE
Update changelog and pubspec

### PR DESCRIPTION
**Pull request by tchabot22**
Bumped GoogleAtlas version and added content to changelog.  Updated pubspec.yaml to reflect latest versions

### CODE REVIEW

File: packages/google_atlas/pubspec.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  atlas: ^0.0.5
+  atlas: ^0.0.5
```
Comment (felangel) [inline packages/google_atlas/pubspec.yaml]: nit remove extra trailing space

File: example/pubspec.yaml [hunk 2]
```diff
@@ -9,8 +9,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  google_atlas:
-    path: ../packages/google_atlas
+  google_atlas: ^0.0.6
```
Comment (felangel) [inline example/pubspec.yaml]: I think we can revert this change so that the example always uses the local copy

**Review by felangel — APPROVED** [review]
lgtm

**Review by noahmateen — APPROVED** [review]
